### DISCUSSION TITLE
Is Support For React Native?

### DISCUSSION THREAD

**Discussion by tuannguyen19296** [jeffreylanters/react-unity-webgl#501] [Question and Answer] [answered]
Hi guys,

Now I use this libary for my React Native project. Could you confirm this can be use in my project?

Thanks!

**Comment by atb-brown** [comment, marked as answer]
https://github.com/jeffreylanters/react-unity-webgl/discussions/492

I'm not an expert on this project, but it looks like it does not work with React-Native on mobile and only works with browsers.
